issue_maker.py

```diff
@@ -36,6 +36,8 @@
 gstr = re.compile(r'(?<=/)[-a-zA-Z0-9]+(?=-lyrics$)')
 # webhook regex
 wdt = re.compile(r'(.+) by (.+) unsupported.')
+# artist and song regex
+asrg = re.compile(r'[A-Z|a-z|\s]+')
```

Comment:
```suggestion
asrg = re.compile(r'[A-Za-z\s]+')
```
Oh, actually `|` does not mean or inside `[]`, `[]` just contain ranges of characters.

Reply:
This is my convention to increase readability (for me). But, since this is not my personal project, I do rewrite this.

Reply:
The thing is, if you keep the | then it will also match `Hello|World` which would be something that it shouldn't match since here we only want to match letters and spaces.

Reply:
OMG I have never thought of that! It seems I have written so many buggy regexes...